submissions/chabuChabu/phpDatabase/viewOrder.php

```diff
@@ -0,0 +1,70 @@
+<?php
+// 1.Connect to database
+require_once("connection.php");
+
+$customer_id  = $_GET['id'];
+
+$savedCookie = 'customerOrder';
+// 2. Check if cookie is empty
+if (empty($_COOKIE[$savedCookie])) {
+
+    include("query.php");
```

Comment:
Instead of including the file when cookie is not set, you have call that function instead

Reply:
@adaugochi great noted